travel_platform/Html/main.html

```diff
@@ -1,111 +1,127 @@
-        <img class="img-responsive" src="pics/entertainment.jpg" alt="entertainment">
-          <span class="text-center">Entertainment</span></a>
-      </div>
-
-      <div class="col-md-4 choose" id="hotels"><a href="https://lviv.itea.ua/">
-        <img class="img-responsive" src="pics/hotel.jpg" alt="hotel">
-          <span class="text-center">Hotels</span></a>
-      </div>
-
-      <div class="col-md-4 choose" id="transport"><a href="https://lviv.itea.ua/">
-        <img class="img-responsive" src="pics/bus.jpg" alt="bus">
-          <span class="text-center">Transport</span></a>
-      </div>
-
-      <div class="col-md-4 choose" id="restaurants"><a href="https://lviv.itea.ua/">
-        <img class="img-responsive" src="pics/restaurant.jpg" alt="restaurant">
-          <span class="text-center">Restaurants</span></a>
-      </div>
+    <div class="col-md-4 choose" id="entertainment"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/entertainment.jpg" alt="entertainment">
+      <span class="text-center">Entertainment</span></a>
     </div>
-  </div>
 
-  <div class="row">
-      <div class="col-md-6 text-center"><h2>How to get there?</h2>
-      </div>
+    <div class="col-md-4 choose" id="hotels"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/hotel.jpg" alt="hotel">
+      <span class="text-center">Hotels</span></a>
+    </div>
 
-      <form method="GET">
-        <div class="col-md-4">
-          <h3><input class="form-control text-center" id="location" type="text" value="enter your location"></h3>
-        </div>
+    <div class="col-md-4 choose" id="transport"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/bus.jpg" alt="bus">
+      <span class="text-center">Transport</span></a>
+    </div>
 
-        <div class="col-md-2">
-          <button type="submit" class="btn btn-default"><h3>Submit</h3></button>
-        </div>
-      </form>
+    <div class="col-md-4 choose" id="restaurants"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/restaurant.jpg" alt="restaurant">
+      <span class="text-center">Restaurants</span></a>
+    </div>
   </div>
+</div>
 
-  <div class="row">
-    <div class="text-right col-md-2"><h3>I have</h3>
-    </div>
-    <form method="GET">
-      <div class="col-md-2">
-          <h3><input class="form-control text-center" id="money" type="text" value="1000"></h3>
-      </div>
-      <div class="col-md-3"><h3>dollars. Where can I</h3>
-      </div>
-      <div class="col-md-1">
-        <button type="submit" class="btn btn-default">Go</button>
-      </div>
-    </form>
-    <div class="col-md-1"><h3>?</h3>
-    </div>
+<div class="row">
+  <div class="col-md-6 text-center"><h2>How to get there?</h2>
   </div>
 
-  <div class="row">
-    <div class="col-md-12 text-center"><h2>Choose a transport</h2>
+  <!-- TODO roman: It would be great to wrap this form by separate div.
```

Comment:
@RomanPryima , FYI. I suggest to merge how it is and fix this as you'll have a free time or you'll change something in this file.